src/testclient/scripts/testSet.env

```
@@ -273,6 +275,25 @@ KTcloudUSwest=$((++IY))		    	# Location: LA, US
 
 
 
+
+
+# IBM-VPC (Total: n Regions / Recommend: n Regions)
+NumRegion[$IndexIBMVPC]=1
+
+TotalNumRegion[$IndexIBMVPC]=5
```

```suggestion
TotalNumRegion[$IndexIBMVPC]=9
```
동작 확인을 해볼 필요는 있겠지만.. ^^